tests/profiling/collector/test_task.py

```diff
@@ -1,31 +1,30 @@
-import os
 import threading
 
 import pytest
 
 from ddtrace.internal import compat
-from ddtrace.internal import nogevent
 from ddtrace.profiling.collector import _task
 
 
-TESTING_GEVENT = os.getenv("DD_PROFILE_TEST_GEVENT", False)
-
-
 def test_get_task_main():
     # type: (...) -> None
     if _task._gevent_tracer is None:
-        assert _task.get_task(nogevent.main_thread_id) == (None, None, None)
-    else:
-        assert _task.get_task(nogevent.main_thread_id) == (compat.main_thread.ident, "MainThread", None)
+        assert _task.get_task(compat.main_thread.ident) == (None, None, None)
 
 
-@pytest.mark.skipif(TESTING_GEVENT, reason="only works without gevent")
 def test_list_tasks_nogevent():
-    assert _task.list_tasks(nogevent.main_thread_id) == []
+    assert _task.list_tasks(compat.main_thread.ident) == []
 
 
-@pytest.mark.skipif(not TESTING_GEVENT, reason="only works with gevent")
```

Comment:
I believe we do need this check `TESTING_GEVENT`, since our testing scheme only installs gevent if this flag is set to true.
https://github.com/DataDog/dd-trace-py/blob/1.x/tox.ini#L75-L89

Reply:
Ah, that answers a question I had. Thanks.

Reply:
I'd be more inclined to add `gevent` to the profiling env deps and remove any gevent special-casing if possible. The whole point of this change is to ensure that we don't have to do anything special for any type of applications. WDYT? @Yun-Kim @emmettbutler

Reply:
I think that's a great idea @P403n1x87

Reply:
This sounds good to me 👍